gradle.properties

```diff
@@ -7,7 +7,7 @@ cordaProductVersion = 5.0.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 179
+cordaApiRevision = 181
```

Comment:
Do we really need to increase the api version here? I think this is only for breaking changes which require a second PR on runtime-os

Reply:
the version compatibility PR gate was failing for this without the version bump

Reply:
Passes without it now, removed